src/main/groovy/com/avast/gradle/dockercompose/ComposeSettings.groovy

```diff
@@ -63,6 +65,9 @@ class ComposeSettings {
     String dockerComposeWorkingDirectory = null
     Duration dockerComposeStopTimeout = Duration.ofSeconds(10)
 
+    File composeLogFile = null
```

Comment:
Now, we have `captureContainersOutputToFile` and `composeLogFile` properties that are similar but the naming is different. Please rename one of these to make it consistent.

Reply:
Ah, and `containerLogDir` as well.